dockerfile

```diff
@@ -0,0 +1,15 @@
+ROM node:10.16.3
```

Comment:
try to correct this, It's FROM and not ROM

Reply:
I fixed it